sdk/spring/spring-reference.yml

```diff
@@ -1,528 +1,754 @@
-      msdocs: https://docs.microsoft.com/azure/storage/blobs/
+      description: |-
+        Azure Blob Storage is Microsoft's object storage solution for the cloud. Blob Storage
+        is optimized for storing massive amounts of unstructured data. Unstructured data is
+        data that does not adhere to a particular data model or definition, such as text or
+        binary data.
+      msDocs: https://docs.microsoft.com/azure/storage/blobs/
       clientSource:
         groupId: com.azure
         artifactId: azure-storage-blob
       artifacts:
-        - artifactId: azure-spring-boot-starter-storage
+        - artifactId: spring-cloud-azure-starter-storage-blob
           groupId: com.azure.spring
           versionGA: 4.0.0
-          versionPreview: 3.0.0-beta.1
-          description: Microsoft's Spring Boot Starter implements Spring Resource abstraction for Azure Storage service which allows you to interact with Azure Blob storage using Spring programming model.
+          versionPreview: 4.0.0-beta.3
+          description: |-
+            Microsoft's Spring Boot Starter helps developers to finish the auto-configuration of
+            Azure Storage Blob, and implements Spring Resource abstraction for Azure Storage
+            service which allows you to interact with Storage Blob using Spring programming model.
           type: spring
           links:
-            repopath: https://search.maven.org/artifact/com.azure.spring/azure-spring-boot-starter-storage
+            github: https://github.com/Azure/azure-sdk-for-java/tree/spring-cloud-azure_4.0.0/sdk/spring/spring-cloud-azure-starter-storage-blob
             msdocs: https://docs.microsoft.com/azure/developer/java/spring-framework/configure-spring-boot-starter-java-app-with-azure-storage
-            github: https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/spring/azure-spring-boot-starter-storage
-            sample: https://github.com/Azure-Samples/azure-spring-boot-samples/tree/main/storage/azure-spring-boot-starter-storage
+            repopath: https://search.maven.org/artifact/com.azure.spring/spring-cloud-azure-starter-storage-blob
+            sample: https://github.com/Azure-Samples/azure-spring-boot-samples/tree/spring-cloud-azure_4.0.0/storage/spring-cloud-azure-starter-storage-blob/
+          springProperties:
+            starter: true
+            bom: spring-cloud-azure-dependencies
+            compatibilityRange: "[2.5.0,2.6.5]"
+            mappings:
+              - compatibilityRange: "[2.5.0,2.5.11]"
+                groupId: com.azure.spring
+                artifactId: spring-cloud-azure-starter-storage-blob
+                version: 4.0.0
+              - compatibilityRange: "[2.6.0,2.6.5]"
+                groupId: com.azure.spring
+                artifactId: spring-cloud-azure-starter-storage-blob
+                version: 4.0.0
+    - name: Storage - File Shares
```

Comment:
The feature name here should be "Storage - Filess Shares", the rule here is to keep consistent with the display name of "azure-storage-file-share" in https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/java-packages.csv, that's the way how the tooling team define categories.